roaring/roaring.go

```diff
@@ -3268,6 +3269,34 @@ func xorBitmapRun(a, b *container) *container {
 	return output
 }
 
+func BitmapsEqual(b, c *Bitmap) error {
+	if b.OpWriter != c.OpWriter {
+		return errors.New("opWriters not equal")
+	}
+	if b.opN != c.opN {
+		return errors.New("opNs not equal")
+	}
+
+	biter, _ := b.Containers.Iterator(0)
+	citer, _ := c.Containers.Iterator(0)
+	bn, cn := biter.Next(), citer.Next()
+	for ; bn && cn; bn, cn = biter.Next(), citer.Next() {
+		bk, bc := biter.Value()
+		ck, cc := citer.Value()
+		if bk != ck {
+			return errors.New("keys not equal")
+		}
+		if !reflect.DeepEqual(bc, cc) {
```

Comment:
I may have written this code, but the use of reflect here makes me uncomfortable - I think we should write a `func (c *Container) Equals(c2 *Container) bool`